index.html

```diff
@@ -165,23 +165,96 @@ <h1>What is a Hackathon?</h1>
         <h1>Schedule</h1>
         <p>Times are listed in your local time zone.</p>
         <div id="schedule-block">
-            <div id="schedule-tapes">
-                <div class="tape"></div>
-                <div class="tape"></div>
+
+            <!-- Binder rings -->
+            <div id="binder-container">
+                <div class="binder-rings">
+                    <div class="binder-ring"></div>
+                    <div class="binder-ring"></div>
+                </div>
+                <div class="binder-rings">
+                    <div class="binder-ring"></div>
+                    <div class="binder-ring"></div>
+                </div>
             </div>
-            <div id="schedule-content">
-                <div id="days">
-                    <div class="day day-first day-active">
-                        <h2 class="number">20</h2>
-                        <h6 class="name">SATURDAY</h6>
+
+            <div id="schedule-tabs">
+                <div class="schedule-tab pre-event schedule-tab-active">
+                    <p>Pre-Event</p>
+                </div>
+                <div class="schedule-tab feb-20">
+                    <p>Feb 20th</p>
+                </div>
+                <div class="schedule-tab feb-21">
+                    <p>Feb 21st</p>
+                </div>
+            </div>
+
+            <div id="schedule-content-wrapper">
+                <div id="pre-event-content" class="schedule-content">
+                    <div class="day-row pre-16">
+                        <div class="day">
+                            <h2 class="number">16</h2>
+                            <h6 class="name">TUESDAY</h6>
```

Comment:
[MDN's usage notes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#usage_notes) say to avoid skipping heading levels but I'm willing to let it pass

Reply:
SJ "Ill allow it" Miller